### PR TITLE
Portfolio details popup window

### PR DESCRIPTION
**Pull request by LethaboMot**
in the project we:

Added the modal.js file which contains the element for the popup
Added CSS to style the desktop and  mobile versions of the popup

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,7 +71,7 @@ <h1>
     <section id="wrap">
         <div class="whitebox">
             <div class="imageclass">
-                <img src="images/Nature.png" width="500" height="400" />
+                <img src="images/Snapshoot Portfolio.svg" width="500" height="400" />
             </div>
             <div class="description">
                 <h2 class="heading">Tonic</h2>
```
[the remark this answers is not in the document]
Comment (LethaboMot) [reply index.html:77]: I thought the popup cards should be created dynamically.

File: style.css [hunk 2]
```diff
@@ -764,7 +764,8 @@ input[type="message"] {
   input[type="message"] {
     border-radius: 2px;
   }
-    /* desktop popup */
+
+  /* desktop popup */
 
   .images-center img {
     display: grid;
```
Comment (NahnahAJ) [inline style.css:771]: - [ ] Please ensure that you follow the Figma Popup design as much as possible. From the screenshot below, you can see that your design has a few spacing issues, and aside from that, the font color used for the `BACK END DEV` and `2015` are different from the Figma design. Kindly fix  this as well :wink: 
- [ ] KIndly do same for the desktop Popup as well :100: 
- Your Design
image

- The Figma Design
image

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
Hi @LethaboMot and @enis-memic , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
- Linters rightly set up and passing :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Good commit messages :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by NahnahAJ — APPROVED** [review]
Hi @LethaboMot and @enis-memic , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- No linter errors :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- Professional README :100: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**